### ISSUE TITLE
Candlestick chart not supported on desktop

### ISSUE THREAD

**Issue by luckysori** [get10101/10101#2533]
![image](https://github.com/get10101/10101/assets/9418575/9e7bb607-5d97-4c0d-8505-7ab1f3323337)

**Comment by Restioson** [comment]
This is known - unfortunately flutter doesn't support webviews on desktop.

**Comment by luckysori** [comment]
> This is known (see [#2123 (comment)](https://github.com/get10101/10101/issues/2123#issuecomment-2060465126)) - unfortunately flutter doesn't support webviews on desktop.

Fair enough. Probably low priority to fix, but I thought it would be good to document it. We have at least some users on desktop.

**Comment by Restioson** [comment]
> We have at least some users on desktop.

Ah, interesting - I was not aware of this... we could fall back to the old implementation, but it was quite complex by comparison

**Comment by bonomat** [comment]
Afaik "Desktop" should be supported because we only support desktop for M1s which is iOS no? 

Eitherway, we are now using a library which supports it. Here is a short how-to: https://inappwebview.dev/docs/intro#setup-macos

**Comment by Restioson** [comment]
From what I can tell, inappwebview does not support desktop in general:
![image](https://github.com/get10101/10101/assets/6688948/e3d520df-5e84-478f-87ca-0ae59f590497)
It does seem to support MacOS though
